docs/multi_process.md

```diff
@@ -0,0 +1,128 @@
+# Using JAX in multi-host and multi-process environments
+
+(test-ref)=
+## Introduction
+
+This guide explains how to use JAX in environments such as [Cloud
+TPU](https://cloud.google.com/tpu) pods where accelerators are spread across
+multiple CPU hosts or JAX processes. We’ll refer to these as “multi-process”
+environments.
+
+This guide specifically focuses on how to use collective communication
+operations (e.g. {func}`jax.lax.psum`) in multi-process settings, although other
+communication methods may be useful too depending on your use case (e.g. RPC,
+[mpi4jax](https://github.com/mpi4jax/mpi4jax)). If you’re not already familiar
+with JAX’s collective operations, we recommend starting with the
+{doc}`/jax-101/06-parallelism` notebook. Note that an important feature of
+multi-process environments is direct communication links between accelerators,
+e.g. the high-speed interconnects for Cloud TPUs or
+[NCCL](https://developer.nvidia.com/nccl) for GPUs. This is what allows
+collective operations to run across multiple process’ worth of accelerators.
+
+
+## Multi-process programming model
+
+Key concepts:
+*   All JAX processes must be manually run.
+*   Each process has a distinct set of _local_ devices it can address. The
+    _global_ devices are the set of all devices across all processes.
+*   Use standard JAX parallelism APIs like {func}`~jax.pmap` and
+    {func}`~jax.experimental.maps.xmap`. Each process “sees” _local_ input and
+    output to parallelized functions, but communication inside the computations
+    is _global_.
+*   Make sure all processes run the same parallel computations in the same
+    order.
+
+
+### Launching JAX processes
+
+Unlike other distributed systems where a single controller node manages many
+worker nodes, JAX uses a “multi-controller” programming model where each JAX
+Python process runs independently, sometimes referred to as an [Single Program,
+Multiple Data (SPMD)](https://en.wikipedia.org/wiki/SPMD) model. Generally, the
```

Comment:
Since "SPMD" is in the glossary, you can use ``` {term}`SPMD` ``` to link there: https://jax.readthedocs.io/en/latest/glossary.html#term-SPMD

There might be some other terms in this guide as well that could be linked or added there

Reply:
Didn't know this, cool! I just changed this one for now.